packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -628,6 +634,9 @@ export function mixinAttach<
 				objectStorage: new MockStorage(),
 			};
 			clientA.channel.connect(services);
+
+			state.containerRuntimeFactory.processAllMessages();
```

Comment:
This is part of a series of changes I made in this file trying to get fuzz tests passing in haste. The is a kludge to get all IdCreationRanges finalized in all containers before connecting a new client.